xml/System.Collections.Concurrent/ConcurrentDictionary`2.xml

```diff
@@ -2704,11 +2704,17 @@ This member is an explicit interface member implementation. It can be used only
         </Parameter>
       </Parameters>
       <Docs>
-        <typeparam name="TAlternateKey">To be added.</typeparam>
-        <param name="lookup">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="TAlternateKey">The alternate type of a key for performing lookups.</typeparam>
+        <param name="lookup">The created lookup instance when the method returns true, or a default instance that should not be used if the method returns false.</param>
+        <summary>
+          Gets an instance of a type that may be used to perform operations on a <see cref="T:System.Collections.Concurrent.ConcurrentDictionary`2"/> 
+          using a <typeparamref name="TAlternateKey"/> as a key instead of a <typeparamref name="TKey"/>.
+        </summary>
+        <returns>true if a lookup could be created; otherwise, false.</returns>
+        <remarks>
+          This instance must be using a comparer that implements <see cref="T:System.Collections.Generic.IAlternateEqualityComparer`2" /> with 
```

Comment:
Did you add this comment manually? I am wondering why this newline was added. The latest version of the tool is supposed to skip these newlines in the middle of sentences.

Reply:
I copied it over manually from the source, since it was being skipped by the tool.

Reply:
OK I'll investigate.